internal/pkg/cli/flag.go

```diff
@@ -176,6 +177,16 @@ Cannot be specified with --%s.`, imageFlag)
 Cannot be specified with --%s.`, imageFlag)
 	storageTypeFlagDescription = fmt.Sprintf(`Type of storage to add. Must be one of:
 %s.`, strings.Join(applyAll(storageTypes, strconv.Quote), ", "))
+	storageLifecycleFlagDescription = fmt.Sprintf(`Whether the storage should be created and deleted
```

Comment:
```suggestion
	storageLifecycleFlagDescription = fmt.Sprintf(`Whether the storage resource should be created and deleted
```

Reply:
actually, is it important that we have "resource" 😢 ? without the resource, the help menu renders so nicely

Reply:
<img width="574" alt="Screenshot 2023-02-17 at 3 50 44 PM" src="https://user-images.githubusercontent.com/79273084/219800559-a985418b-d249-4b45-a32d-1c5e24e52cbe.png">
that all 3 of these rows are filled

Reply:
Nah, it's fine without, I think. :)